geomstats/learning/frechet_mean.py

```diff
@@ -333,6 +333,50 @@ def _adaptive_gradient_descent(
 
     return current_mean
 
+def _circle_mean (points):
+    """Determine the mean on a circle.
+    
+    Data are expected in radians in the range [-pi, pi). The mean is returned
+    in the same range. If the mean is unique, this algorithm is guaranteed to
+    find it. It is not vulnerable to local minima of the Frechet function. If
+    the mean is not unique, the algorithm only returns one of the means. Which
+    mean is returned depends on numerical rounding errors.
+    
+    Reference
+    ---------
+    Hotz, T. and S. F. Huckemann (2015), "Intrinsic means on the circle:
```

Comment:
Can you use this syntax to provide the references?
https://github.com/geomstats/geomstats/blob/048a99dc9dff3f86e42025264ac0811ed6888f7c/geomstats/geometry/lie_algebra.py#L62

It allows this part of the docstring to be automatically parsed and render nicely for our documentation website:
https://geomstats.github.io/geometry.html#module-geomstats.geometry.lie_algebra